File: sdkwork-app-sdk-csharp/Models/CouponVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class CouponVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? CouponId { get; set; }
        public string? Name { get; set; }
        public string? Type { get; set; }
        public string? TypeName { get; set; }
        public string? Description { get; set; }
        public int? Amount { get; set; }
        public int? PointCost { get; set; }
        public double? Discount { get; set; }
        public int? MinConsume { get; set; }
        public string? StartTime { get; set; }
        public string? EndTime { get; set; }
        public int? Total { get; set; }
        public int? GetLimit { get; set; }
        public int? ReceivedCount { get; set; }
        public int? UsedCount { get; set; }
        public string? Status { get; set; }
        public string? StatusName { get; set; }
        public bool? Stackable { get; set; }
        public string? ScopeType { get; set; }
        public string? ScopeValue { get; set; }
        public int? RemainingCount { get; set; }
        public bool? CanReceive { get; set; }
        public bool? PointsExchange { get; set; }
    }
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/CouponVO.kt
package com.sdkwork.app

data class CouponVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val couponId: String? = null,
    val name: String? = null,
    val type: String? = null,
    val typeName: String? = null,
    val description: String? = null,
    val amount: Int? = null,
    val pointCost: Int? = null,
    val discount: Double? = null,
    val minConsume: Int? = null,
    val startTime: String? = null,
    val endTime: String? = null,
    val total: Int? = null,
    val getLimit: Int? = null,
    val receivedCount: Int? = null,
    val usedCount: Int? = null,
    val status: String? = null,
    val statusName: String? = null,
    val stackable: Boolean? = null,
    val scopeType: String? = null,
    val scopeValue: String? = null,
    val remainingCount: Int? = null,
    val canReceive: Boolean? = null,
    val pointsExchange: Boolean? = null
)
